README.md

```diff
@@ -46,20 +46,20 @@ Note that the rule is namespaced with `plugin/`.
 
 Like so:
 
-```js
+```json
 // .stylelintrc
 {
 	"plugins": [
-		"/path/to/stylelint-no-browser-hacks/lib"
+		"stylelint-no-browser-hacks"
```

Comment:
This is valid only when this packageis installed locally, but if it's installed globally you must specify an absolute path.

Reply:
Yeap! Let's better add `-D` flag to installation command, this is most common.
If a user uses stylelint globally (without global plugins), stylelint will provide an error to set up `configDir`.

Reply:
I'm not an expert of Stylelint: may you point me the doc for such flag?

Reply:
I mean `npm i stylelint-no-browser-hacks -D`